plenum/test/replica_removing/test_replica_removing.py

```diff
@@ -102,22 +102,31 @@ def test_ordered_request_freed_on_replica_removal(looper,
                                                   chkFreqPatched,
                                                   view_change):
     node = txnPoolNodeSet[0]
-    sdk_send_random_and_check(looper, txnPoolNodeSet, sdk_pool_handle, sdk_wallet_client, 1)
+    if node.master_replica._consensus_data.checkpoints:
+        num_requests_to_stable_checkpoint = node.master_replica._consensus_data.stable_checkpoint - \
```

Comment:
Since `consensus_data.checkpoints` contains checkpoints sent by us, all Checkpoint's `seqNoEnd` there are greater or equal to `consensus_data.stable_checkpoint`. So, what do we need this code, and why don't it return a value <= 0?